internal/commands/app/create_inputs_test.go

```diff
@@ -111,7 +112,8 @@ func TestAppCreateInputsResolveName(t *testing.T) {
 				return []realm.App{testApp}, nil
 			}
 
-			err := tc.inputs.resolveName(nil, rc, tc.appRemote)
+			err := tc.inputs.resolveName(nil, rc, realm.AppFilter{
+				GroupID: tc.appRemote.GroupID, App: tc.appRemote.AppID})
```

Comment:
same style nit throughout this file, its fairly common in Go to have that trailing `)` on its own newline (if/when you put each arg/field on its own newline).  i think here you can go one of two routes:

```go
err := tc.inputs.resolveName(nil, rc, realm.AppFilter{tc.appRemote.GroupID, tc.appRemote.AppID})
```

or

```go
err := tc.inputs.resolveName(nil, rc, realm.AppFilter{
  GroupID: tc.appRemote.GroupID,
  App: tc.appRemote.AppID,
})
```

that being said, depending on how we refactor the function signature for `resolveName` and/or what we do with the return value from `resolveRemoteApp`, maybe we can just edit the `tc` struct here (`appRemote` could be split into `groupID string` and `appID string` and/or changed to `realm.App`